### DISCUSSION TITLE
FEATURE REQUEST: Please add a feature to archive read articles for future reference

### DISCUSSION THREAD

**Discussion by Zidan2k9** [dailydotdev/daily#617] [Feature Requests]
the extension has a reading history section and bookmarks. But bookmarks are sometimes used for reading articles at a later time.

If a user was able to archive an article they have read but want to save it for future reference (and return to it 2 or 3 months in the future), it would be easier to find then scrolling through reading history.

**Comment by idoshamun** [comment]
Hi @Zidan2k9,

I think the "read it later" feature should address your concern. This way you can use bookmarks for long-term references.
Please see the discussion here: https://github.com/dailydotdev/daily/discussions/364
